src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -2895,4 +2895,88 @@ public void startFakeRepairSchedule() {
           Response.Status.NOT_FOUND);
     });
   }
+
+  @And("^I add 11 and abort the most recent 10 repairs for cluster \"([^\"]*)\" and keyspace \"([^\"]*)\"$")
+  public void addAndAbortRepairs(String clusterName, String keyspace) throws Throwable {
```

Comment:
`11` and `10` should be variables, so that we can reuse this step if needed with different numbers.

```suggestion
  @And("^I add (\\d+) and abort the most recent (\\d+) repairs for cluster \"([^\"]*)\" and keyspace \"([^\"]*)\"$")
  public void addAndAbortRepairs(int nbRepairRuns, int abortedRepairRuns, String clusterName, String keyspace) throws Throwable {
```

Reply:
I've made the requested changes. Thinking on this, should this not be several steps? So perhaps the spec should be:
* When I add 11 repairs
* When I set the state on the most recent 10 repairs to ABORTED

That would better promote reusability I think.